fido-uaf-core/src/main/java/com/nexenio/fido/uaf/core/message/MatchCriteria.java

```diff
@@ -34,4 +34,34 @@
     @JsonProperty("aaid")
     private List<String> aaids = new ArrayList<>();
 
+    @SerializedName("vendorID")
+    @JsonProperty("vendorID")
+    private List<String> vendorIds = new ArrayList<>();
+
+    @SerializedName("keyIDs")
+    @JsonProperty("keyIDs")
+    private List<String> keyIds = new ArrayList<>();
+
+    private long userVerification;
+
+    private int keyProtection;
+
+    private int matcherProtection;
+
+    private long attachmentHint;
+
+    private int tcDisplay;
```

Comment:
Before looking it up i didn't know that this stands for `transaction confirmation display`. Maybe we can rename that.